pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -374,11 +374,19 @@ private class ListCmd extends CliCommand {
                 "--bundle" }, description = "Namespace bundle to get list of topics")
         private String bundle;
 
+        @Parameter(names = { "-wst",
+                "--with-system-topic" }, description = "Include system topic")
```

Comment:
`--include-system-topic` is more descriptive?

Reply:
yes, fixed.